content/en/docs/reference/kubectl/overview.md

```diff
@@ -373,6 +373,15 @@ kubectl logs <pod-name>
 kubectl logs -f <pod-name>
 ```
 
+`kubectl diff` - Compares the current state of the cluster against the state that the cluster would be in if the manifest was applied.
+
+```shell
+# Diff resources included in pod.json.
+kubectl diff -f pod.json
```

Comment:
nit: add a new line to keep same style with other kubectl command examples should be better.

Reply:
Ok, Thanks!

Reply:
@zhouya0 , what do you think about shortening the description. Here are a few ideas:
Compare the changes to a cluster before applying the changes.
View a diff of the changes to a cluster before updating the cluster.
Display the changes to a cluster before applying the changes.

Reply:
> Compare the changes to a cluster **before applying the changes.**

that sounds a little bit like that command will also apply the changes?

Reply:
OK, I agree. 
View a diff of the proposed updates to a cluster.
Compare cluster changes to the current state of your cluster.

Reply:
> View a diff of the proposed updates to a cluster.

I really like this one, thanks for the suggestions!

Reply:
@kbhawkey  I agree with `View a diff of the proposed updates to a cluster.` , thanks!